core/pkg/resultshandling/printer/v1/jsonprinter.go

```diff
@@ -42,4 +42,6 @@ func (jsonPrinter *JsonPrinter) ActionPrint(opaSessionObj *cautils.OPASessionObj
 		logger.L().Fatal("failed to convert posture report object")
 	}
 	jsonPrinter.writer.Write(postureReportStr)
```

Comment:
We should handle the error here the same way we handled in other places

Reply:
Done!!